examples/crm/src/sales/SalesCreate.tsx

```diff
@@ -0,0 +1,13 @@
+import { Create, PasswordInput, required, SimpleForm } from 'react-admin';
+import { SalesForm } from './SalesForm';
+
+export function SalesCreate() {
+    return (
+        <Create>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏We redirect on edit page after creation, it's on purpose ?